src/node/signature_aggregator.rs

```diff
@@ -0,0 +1,302 @@
+// Copyright 2021 MaidSafe.net limited.
+//
+// This SAFE Network Software is licensed to you under the MIT license <LICENSE-MIT
+// http://opensource.org/licenses/MIT> or the Modified BSD license <LICENSE-BSD
+// https://opensource.org/licenses/BSD-3-Clause>, at your option. This file may not be copied,
+// modified, or distributed except according to those terms. Please review the Licences for the
+// specific language governing permissions and limitations relating to use of the SAFE Network
+// Software.
+
+use super::signed::{Signed, SignedShare};
+use std::{
+    collections::HashMap,
+    fmt::Debug,
+    time::{Duration, Instant},
+};
+use thiserror::Error;
+use threshold_crypto as bls;
+use tiny_keccak::{Hasher, Sha3};
+
+/// Default duration since their last modification after which all unaggregated entries expire.
+pub const DEFAULT_EXPIRATION: Duration = Duration::from_secs(120);
+
```

Comment:
Seeing this file now @maqi  I wonder as clients also need this part should it actually be just the sig_aggregator crate? Sorry for mixed messages but perhaps this tiny crate is useful with just this in it?